src/statsd_client.cpp

```diff
@@ -102,11 +102,34 @@ void StatsdClient::config(const string& host, int port, const string& ns)
     d->port = port;
     d->init = false;
     if ( d->sock >= 0 ) {
-        close(d->sock);
+#ifdef WIN32
+        int ret = closesocket(d->sock);
+#else
+        int ret = close(d->sock);
+#endif
     }
     d->sock = -1;
 }
 
+#ifdef WIN32
+int inet_aton(const char *src, void *dst)
```

Comment:
Maybe it would be better to use `LookupHost` from `netbase.h` instead of `inet_aton`? Advantage would be that we can use hostnames instead of only IP addresses.

Reply:
I assume this can be removed now due to the use of LookupHost